deliver/lib/deliver/upload_metadata.rb

```diff
@@ -400,6 +401,18 @@ def assign_defaults(options)
       end
     end
 
+    def detect_existing_app_store_languages(options)
```

Comment:
Btw is this safe assumption that the `default` is always `primary_locale` in `Spaceship::ConnectAPI::App`? If so, this method can be as easy as just referencing `app.primary_locale`, can't it?

https://github.com/fastlane/fastlane/blob/678966c4175a9e51434dae97a5b147bcb7dfc8d9/spaceship/lib/spaceship/connect_api/models/app.rb#L4-L12

I wasn't sure if this method is only to work out what "default" is though.

Reply:
So "default" in _deliver_ doesn't actually refer to the the primary locale in ASC. The "default" metadata gets used for all languages that are explicitly specified in the metadata but present on ASC. So for example...

## Example 1 (this worked previously)
ASC: en-US, fr-FR, es-MX
deliver meta: en-US, default
👆 default here would get used on fr-FR and es-MX

## Example 2 (this didn't work until this fix)
ASC: en-US, fr-FR, es-MX
deliver meta: default
👆 default here would get used on en-US, fr-FR, and es-MX

Reply:
Ah, that's right. Thank you for the clarification!